### PR TITLE
Consistently format docstrings.

### PR DESCRIPTION
**Pull request by schmmd**
Without the space (`foo: bar`):

image

With the space (`foo : bar`):

image

`find . -name '*py' -exec gsed -i 's/ *: ``/ : ``/g' \{\} \;`

### CODE REVIEW

**Comment by schmmd** [comment]
@hayata-yamamoto FYI--I finally got to the bottom of the "space or no space" question!

**Comment by schmmd** [comment]
Two files threw exceptions, which I didn't investigate much.  I simply reverted the syntax.

```
./scripts/verify.py --checks build-docs                                                                                                                                                                              (allennlp) 
Verifying with ['build-docs']
Documentation (build):
make check
rm -rf _build
sphinx-build -W -T -b html -d _build/doctrees   . _build/html
Running Sphinx v2.3.1
loading translations [en]... done
making output directory... done
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 147 source files that are out of date
updating environment: [new config] 147 added, 0 changed, 0 removed
reading sources... [100%] index                                                                                                                                                                                                                                               

Traceback (most recent call last):
  File "/Users/michael/miniconda3/envs/allennlp/lib/python3.7/site-packages/sphinx/cmd/build.py", line 276, in build_main
    app.build(args.force_all, filenames)
  File "/Users/michael/miniconda3/envs/allennlp/lib/python3.7/site-packages/sphinx/application.py", line 353, in build
    self.builder.build_update()
  File "/Users/michael/miniconda3/envs/allennlp/lib/python3.7/site-packages/sphinx/builders/__init__.py", line 299, in build_update
    len(to_build))
  File "/Users/michael/miniconda3/envs/allennlp/lib/python3.7/site-packages/sphinx/builders/__init__.py", line 311, in build
    updated_docnames = set(self.read())
  File "/Users/michael/miniconda3/envs/allennlp/lib/python3.7/contextlib.py", line 119, in __exit__
    next(self.gen)
  File "/Users/michael/miniconda3/envs/allennlp/lib/python3.7/site-packages/sphinx/util/logging.py", line 219, in pending_warnings
    memhandler.flushTo(logger)
  File "/Users/michael/miniconda3/envs/allennlp/lib/python3.7/site-packages/sphinx/util/logging.py", line 184, in flushTo
    logger.handle(record)
  File "/Users/michael/miniconda3/envs/allennlp/lib/python3.7/logging/__init__.py", line 1524, in handle
    self.callHandlers(record)
  File "/Users/michael/miniconda3/envs/allennlp/lib/python3.7/logging/__init__.py", line 1586, in callHandlers
    hdlr.handle(record)
  File "/Users/michael/miniconda3/envs/allennlp/lib/python3.7/logging/__init__.py", line 890, in handle
    rv = self.filter(record)
  File "/Users/michael/miniconda3/envs/allennlp/lib/python3.7/logging/__init__.py", line 751, in filter
    result = f.filter(record)
  File "/Users/michael/miniconda3/envs/allennlp/lib/python3.7/site-packages/sphinx/util/logging.py", line 404, in filter
    raise SphinxWarning(location + ":" + message)
sphinx.errors.SphinxWarning: /Users/michael/hack/allenai/allennlp/allennlp/data/dataset_readers/sequence_tagging.py:docstring of allennlp.data.dataset_readers.sequence_tagging.SequenceTaggingDatasetReader:12:Inline literal start-string without end-string.

Warning, treated as error:
/Users/michael/hack/allenai/allennlp/allennlp/data/dataset_readers/sequence_tagging.py:docstring of allennlp.data.dataset_readers.sequence_tagging.SequenceTaggingDatasetReader:12:Inline literal start-string without end-string.
make[1]: *** [check] Error 2
make: *** [html-strict] Error 2
```

**Comment by DeNeutoy** [comment]
@schmmd FYI I have a more comprehensive fix for this (new docs), so i'd advise you to not do any more work on this afterward.

File: allennlp/common/registrable.py [hunk 1]
```diff
@@ -48,9 +48,9 @@ def register(cls: Type[T], name: str, exist_ok=False):
 
         Parameters
         ----------
-        name : ``str``
+        name: ``str``
             The name to register the class under.
-        exist_ok : ``bool`, optional (default=False)
+        exist_ok: ``bool`, optional (default=False)
```
Comment (matt-gardner) [inline allennlp/common/registrable.py]: You just need to add one more backtick here - it's lopsided, that's why sphinx is complaining.

File: allennlp/data/dataset_readers/sequence_tagging.py [hunk 2]
```diff
@@ -27,9 +27,9 @@ class SequenceTaggingDatasetReader(DatasetReader):
 
     Parameters
     ----------
-    word_tag_delimiter : ``str``, optional (default=``"###"``)
+    word_tag_delimiter: ``str``, optional (default=``"###"``)
```
Comment (matt-gardner) [inline allennlp/data/dataset_readers/sequence_tagging.py]: Not sure what this one is, but it's probably the backticks in the `default=` part.

**Comment by schmmd** [comment]
@DeNeutoy music to my ears... what we have right now is a bit of a mess.

**Comment by schmmd** [comment]
@matt-gardner I'm going to merge this in the name of consistency--that's all I was going for.  This moves everything to how we most often format docstrings, although I accept your criticism that they could look better.

Presently we do either approach, alternating even within the same docstring, which is a bit of a mess.

I can't wait to see what @DeNeutoy is up to.

File: allennlp/common/registrable.py (hunk 1 above)
Comment (schmmd) [reply allennlp/common/registrable.py]: Yep--sorry I missed this comment so I didn't fix this (but it isn't a regression from this PR).

**Pull request merged**